apps/backend/src/deposits/pos-deposit.service.ts

```diff
@@ -22,18 +22,18 @@ export class PosDepositService {
     private posDepositRepo: Repository<POSDepositEntity>
   ) {}
 
-  async findPOSDeposits(
+  async findPosDeposits(
     dateRange: DateRange,
     program: Ministries,
-    location_id: number,
+    location_ids: number[],
     statuses?: MatchStatus[]
   ): Promise<POSDepositEntity[]> {
     const depositStatuses = statuses ? statuses : MatchStatusAll;
     const { minDate, maxDate } = dateRange;
     const locations: LocationEntity[] =
       await this.locationService.getLocationsByID(
```

Comment:
I think we can pass the merchant ids in from the reconcile lambda directly, since we're including those in the original location lookup? Just to save from making this extra query. (I might be wrong though, still reviewing :)

Reply:
To be done in a later PR